src/shared/utils/vis.ts

```diff
@@ -315,54 +315,27 @@ export const getMainColumnName = (
   return ''
 }
 
-const getS2CellID = (table: Table, index: number): string => {
-  const column = table.getColumn('s2_cell_id')
-  if (!column) {
+const getColumnValue = (table: Table, field: string) => {
+  const fieldColumn = table.getColumn('_field')
+
+  if (!fieldColumn) {
     throw new Error(
-      'Cannot retrieve s2_cell_id column - table does not conform to required structure of Table type'
+      'Cannot retrieve _field column - table does not conform to required structure of Table type'
     )
   }
+  const index = fieldColumn.findIndex(val => val === field)
```

Comment:
nit: `val` is kind of vague as a name. not a huge deal since it's kind of clear what's happening but just an area for small improvement